pages/login.html

```diff
@@ -5,80 +5,89 @@
-            <button class="btn-secondary nav-btn btn-small">Sign up</button>
-            <button class="btn-primary nav-btn btn-small">Log in</button>
+            <a href="../pages/cart.html" class=""
+              ><button class="btn-icon nav-icons m-x-1">
+                <i class="fas is-dark fa-shopping-cart"></i></button
+            ></a>
+            <a href="../pages/wishlist.html" class="">
+              <button class="btn-icon nav-icons m-x-1">
+                <i class="fas is-dark fa-heart"></i>
+              </button>
+            </a>
+            <a href="../pages/sign-up.html"
+              ><button class="btn-secondary nav-btn btn-small">
+                Sign up
+              </button></a
+            >
+            <a href="../pages/login.html"
+              ><button class="btn-primary nav-btn btn-small">Log in</button></a
+            >
           </div>
         </div>
       </nav>
     </section>
     <!-- login -->
     <section class="login-section p-x-1">
-        <div class="login b-radius center-x m-up-6 shadow  p-y-2 p-x-4">
-            <div class="textbox">
-                <div class="title"> Login</div>
-            </div>
-            <div class="form-div m-up-1 ">
-                <p class="form-label">Email</p>
-                <input
-                  type="text"
-                  class="form-input input-focused"
-                  placeholder="enter your email id"
-                  required
-                />
-                <p class="form-label m-up-2">Password</p>
-                <input
-                  type="password"
-                  class="form-input input-focused"
-                  placeholder="enter your password"
-                  required
-                />
-                
-              </div>
-              <label for="" class="m-up-2 form-checkbox">
-                <input type="checkbox" class="">Remember me
-            </label>
-            <a href="" class="link-secondary m-l-6">Forgot password</a>
-            <div class="btn-vertical m-up-3 center-text">
-                <button class="btn-primary m-dw-1 btn-small">Login</button>
-                <a href="" class="is- link">Create Account</a>
-            </div>
+      <div class="login b-radius center-x m-up-6 shadow p-y-2 p-x-4">
+        <div class="textbox">
+          <div class="title">Login</div>
+        </div>
+        <div class="form-div m-up-1">
+          <p class="form-label">Email</p>
+          <input
+            type="text"
+            class="form-input input-focused"
+            placeholder="enter your email id"
+            required
+          />
+          <p class="form-label m-up-2">Password</p>
+          <input
+            type="password"
+            class="form-input input-focused"
+            placeholder="enter your password"
+            required
+          />
+        </div>
+        <label for="" class="m-up-2 form-checkbox">
+          <input type="checkbox" class="" />Remember me
```

Comment:
might not need `for` attribute if input is nested within label, else you can link `for` attribute to `id` attribute of input